docs/en/operation/Scalability-Tuning.md

```diff
@@ -139,6 +141,15 @@ ALLUXIO_MASTER_JAVA_OPTS+=" -Xms256g -Xmx256g "
 When setting the heap size, ensure that there is enough memory allocated for off heap storage.
 For example, spawning `4000` threads with a default thread stack size of `1 MB` requires at least
 `4 GB` of off-heap space available.
+* Network buffers are often allocated from a pool of direct memory in Java. 
```

Comment:
Do you know when exactly is buffer allocated on/off heap? I mean it sounds like the buffer is not always off heap.

Reply:
I think netty buffers are allocated from direct memory? i am not entirely sure. @ggezer do you have more context here?